### ISSUE TITLE
Potentially vulnerability using TLS CBC ciphers

### ISSUE THREAD

**Issue by Charles1000Chen** [zhmcclient/zhmc-prometheus-exporter#509]
**Describe the bug**

Potentially vulnerability using TLS CBC ciphers

**Expected behavior**

The test item "LUCKY13" shoud be "OK" in [testssh.sh](https://testssl.sh/) test result.

**To Reproduce**
Test with [testssl.sh](https://testssl.sh/), it will report the issue in its test result.

**Environment information**
* Output of `zhmc_prometheus_exporter --version`:
* HMC version:

**Command output**
```
{
    "id"           : "LUCKY13",
    "severity"     : "LOW",
    "cve"          : "CVE-2013-0169",
    "cwe"          : "CWE-310",
    "finding"      : "potentially vulnerable, uses TLS CBC ciphers"
}

```
**Log file**
<-- If possible, attach a log file generated with '--log-comp all=debug --log exporter.log'. -->

**Comment by andy-maier** [comment]
See #508 , should be solved together.

**Comment by andy-maier** [comment]
The two ciphers are CBC ciphers that are reported by testssl.sh:
```
LUCKY13 (CVE-2013-0169), experimental     potentially VULNERABLE, uses cipher block chaining (CBC) ciphers with TLS. Check patches
```

This was on macOS with:
* Python 3.12.3
* OpenSSL 3.3.0
* testssl.sh 3.0.8

**Comment by andy-maier** [comment]
Solved with PR #551 .

**Comment by andy-maier** [comment]
It turns out that the vendorized copy of prometheus-client in exporter version 1.7.0 does not have the fix.

The fix will be in exporter version 2.4.0 and 2.3.3.
